regcomp.c

```diff
@@ -12826,9 +12827,9 @@ S_regpiece(pTHX_ RExC_state_t *pRExC_state, I32 *flagp, U32 depth)
     }
 
     /* Forbid extra quantifiers */
-    if (ISMULT2(RExC_parse)) {
-        RExC_parse++;
-        vFAIL("Nested quantifiers");
+    if (isQUANTIFIER(RExC_parse, RExC_end)) {
+	RExC_parse++;
+	vFAIL("Nested quantifiers");
```

Comment:
Did you intentionally change the spaces to tabs here?

Reply:
All I can think of is this is a vim bug.  But no; I'll fix it.